frontend/src/lib-customizations/types.d.ts

```diff
@@ -268,6 +268,12 @@ interface BaseFeatureFlags {
    * Show the Titania errors report in the reports list
    */
   titaniaErrorsReport?: boolean
+
+  /**
+   * Select the type of holiday questionnaire
+   * Note the corresponding backend environment variable feature flag.
+   */
+  holidayQuestionnaireType?: 'FIXED_PERIOD' | 'OPEN_RANGES'
```

Comment:
You should not add the same config in both backend and frontend, because they can conflict. You can send the backend value to the frontend in `EmployeeFeatures`, see `SystemController.kt`